src/components/Cart/CartTotals.js

```diff
@@ -51,10 +51,13 @@ export default function CartTotals({ value }) {
         >
           <CustomerConsumer>
             {(ctx) => {
+              let customer = ctx.customers.filter(
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find
`ctx.customers.find(customer => customer.name === ctx.customer)`

btw don't you have `ctx.customer` available already?

Reply:
this find/filter should not be done here, context should provide you the selected customer already

Reply:
plz see my other comment below